app/templates/people/person/positions/position/index.hbs

```diff
@@ -0,0 +1,259 @@
+                          {{@model.mandatory.contacts.firstObject.telefone}}
+                        </p>
+                      </div>
+                    </div>
+                  </div>
+                {{/if}}
+                {{#if
+                  @model.mandatory.contacts.firstObject.address.fullAddress
+                }}
+                  <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
+                    <div class="au-o-grid">
+                      <div class="au-o-grid__item au-u-1-4">
+                        <p><strong>Adres</strong></p>
+                      </div>
+                      <div class="au-o-grid__item au-u-3-4">
+                        <p>
+                          {{@model.mandatory.contacts.firstObject.address.fullAddress}}
+                        </p>
+                      </div>
+                    </div>
+                  </div>
+                {{/if}}
+                {{#if @model.mandatory.contacts.firstObject.telefone}}
+                  <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
+                    <div class="au-o-grid">
+                      <div class="au-o-grid__item au-u-1-4">
+                        <p><strong>GSM nummer</strong></p>
+                      </div>
+                      <div class="au-o-grid__item au-u-3-4">
+                        {{!-- TODO: Detect if it is a landline or mobile number --}}
+                        <p>
+                          {{@model.mandatory.contacts.firstObject.telefone}}
+                        </p>
+                      </div>
+                    </div>
+                  </div>
+                {{/if}}
+              </div>
+            </div>
+          </c.content>
+        </AuCard>
+      </div>
+    {{/if}}
+
+    <div>
+      <AuToolbar class="au-u-margin-bottom-small">
+        <AuToolbarGroup>
+          <AuHeading @level="4" @skin="5">Aanstelling</AuHeading>
+        </AuToolbarGroup>
+      </AuToolbar>
+      <AuCard @standOut={{true}} as |c|>
+        <c.content>
+          <div class="au-flow au-o-flow--large">
+            <div class="au-o-grid">
+              <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
+                <div class="au-o-grid">
+                  <div class="au-o-grid__item au-u-1-4">
+                    <p><strong>Start mandaat</strong></p>
+                  </div>
+                  <div class="au-o-grid__item au-u-3-4">
+                    <p>{{date-format @model.mandatory.startDate}}</p>
+                  </div>
+                </div>
+              </div>
+              <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
+                <div class="au-o-grid">
+                  <div class="au-o-grid__item au-u-1-4">
+                    <p><strong>Bestuursperiode</strong></p>
+                  </div>
+                  <div class="au-o-grid__item au-u-3-4">
+                    <p>{{@model.mandatory.mandate.governingBody.period}}</p>
```

Comment:
This seems to be empty sometimes. Is this the correct path?

Reply:
Also maybe add a check to not show the field if it's empty?

Reply:
Yes, sometimes is empty